data_managers/data_manager_diamond_database_builder/data_manager/data_manager_diamond_database_builder.xml

```diff
@@ -1,10 +1,10 @@
-<tool id="diamond_database_builder" name="Diamond" tool_type="manage_data" version="0.0.1">
+<tool id="diamond_database_builder" name="Diamond" tool_type="manage_data" version="0.0.2">
     <description> Database builder</description>
     <requirements>
-        <requirement type="package" version="0.6.13">diamond</requirement>
+        <requirement type="package" version="0.9.29">diamond</requirement>
```

Comment:
Can you also add python (I guess 2.7 if you don't want to fix data_manager_diamond_database_builder.py) as a dependency ? And in the script it'd be good to replace https://github.com/bgruening/galaxytools/blob/master/data_managers/data_manager_diamond_database_builder/data_manager/data_manager_diamond_database_builder.py#L15 with json.dumps / json.loads